### ISSUE TITLE
Implement Age 10 cards for Unseen Expansion

### ISSUE THREAD

**Issue by ultimatefiend** [micahstairs/bga-innovation#1083]
This issue will track the implementation of all Age 10 cards for the Unseen expansion. Please update this comment before working on any of these cards (if the work is unblocked) so that we don't have duplicate work being done. We no longer need to file a separate issue for each card, since a consolidated list of the cards will keep us more organized.

- [x] 575 : Hacking
- [x] 576 : Inhomogeneous Cosmology
- [x] 577 : Fight Club
- [x] 578 : Secret Santa
- [x] 579 : Cryptocurrency
- [x] 580 : Denver Airport
- [x] 581 : Ride-hailing
- [x] 582 : Whatchamacallit
- [x] 583 : 3D Printing
- [x] 584 : Order of the Occult Hand
